src/Microsoft.PowerShell.Commands.Management/commands/management/ResolvePathCommand.cs

```diff
@@ -112,7 +112,7 @@ protected override void ProcessRecord()
                             if (currentPath.Drive != SessionState.Path.CurrentLocation.Drive &&
                                 !currentPath.ProviderPath.StartsWith(SessionState.Path.CurrentLocation.Drive.Root))
                             {
-                                WriteObject(currentPath.Path, false);
+                                WriteObject(sendToPipeline: currentPath.Path, enumerateCollection: false);
```

Comment:
the parameter names should only be used for literal values like `false` or `true` not when you are using a variable like `currentPath.Path`.